cpp/src/arrow/engine/substrait/expression_internal.cc

```diff
@@ -82,15 +83,6 @@ Result<SubstraitCall> DecodeScalarFunction(
   return std::move(call);
 }
 
-std::string EnumToString(int value, const google::protobuf::EnumDescriptor* descriptor) {
-  const google::protobuf::EnumValueDescriptor* value_desc =
-      descriptor->FindValueByNumber(value);
-  if (value_desc == nullptr) {
-    return "unknown";
-  }
-  return value_desc->name();
-}
-
```

Comment:
I removed this method from `expression_internal.cc` since this can be a generic util. And put it to the `util.cc`